app/src/main/java/com/github/orkest/MainActivity.kt

```diff
@@ -1,17 +1,26 @@
 package com.github.orkest
```

Comment:
Is the signUp incorporated with this signIn or is the signUp with google is yet to be implemented

Reply:
the signup will be in between the two